packages/components/src/search-list-control/style.scss

```diff
@@ -43,12 +43,11 @@
 }
 
 .woocommerce-search-list__list {
+	border: 1px solid $gray-100;
```

Comment:
Before, the list only had top/bottom border but I changed it so it also has left/right border. We have a CSS rule for that in WC Blocks but I think it makes sense upstreaming.